src/parser/jobs/nin/modules/Positionals.ts

```diff
@@ -1,15 +1,24 @@
 import {PotencyModifier, Positionals as CorePositionals} from 'parser/core/modules/Positionals'
 
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+const ARMOR_CRUSH_BASE_POTENCY = 140
+const ARMOR_CRUSH_COMBO_POTENCY_6_0 = 340
+const ARMOR_CRUSH_COMBO_POTENCY_6_1 = 360
+const AEOLIAN_EDGE_BASE_POTENCY = 140
+const AEOLIAN_EDGE_COMBO_POTENCY_6_0 = 360
+const AEOLIAN_EDGE_COMBO_POTENCY_6_1 = 380
+/* eslint-enable @typescript-eslint/no-magic-numbers */
```

Comment:
Couple of things here:
- shouldn't need to disable magic numbers as you're assigning them to consts, making them not magic numbers
- for patch specific fields, we generally squash the patch version to 3 consecutive digits (to handle stuff like 6.15), so in this case it'd be `SKILL_POTENCY_600` and `SKILL_POTENCY_610`, mostly a style thing but that's how we do it